front-end/src/pages/ProposalDetail/ProposalDetail.scss

```diff
@@ -0,0 +1,311 @@
+
+    .description-wraper {
+        margin-top: 90px;
+
+        .MuiFormControl-fullWidth {
+            label.Mui-focused {
+                color: #599C6D;
+            }
+
+            label.MuiFormLabel-root {
+                color: #599C6D;
+            }
+
+            textarea {
+                color: rgba(0, 0, 0, 0.8);
+                font-weight: 400;
+                font-size: 16px;
+                line-height: 24px;
+            }
+
+            .MuiFormHelperText-root {
+                color: #ccc;
+            }
+        }
+
+        .description-helper {
+            margin-top: -35px;
+
+            P {
+                text-align: left;
+            }
+        }
+    }
+
+    .MuiFormHelperText-root {
+        text-align: right;
+        color: red;
+    }
+
+    .MuiDropzoneArea-root {
+        min-height: 157px;
+        max-height: 244px;
+        border: none;
+        background: rgba(34, 123, 60, 0.75);
+        position: relative;
+        outline: none;
+
+        .MuiDropzonePreviewList-root {
+            position: absolute;
+            top: 0;
+            justify-content: center;
+            margin: 0;
+            width: 100%;
+            height: 100%;
+
+            .MuiDropzonePreviewList-imageContainer {
+                padding: 0;
+                position: absolute;
+                max-width: 100%;
+
+                .MuiDropzonePreviewList-image {
+                    height: 156px;
+                    width: 100%;
+                    min-width: 200px;
+                    max-width: 412px;
+                }
+
+                .MuiDropzonePreviewList-removeButton {
+                    top: 2px;
+                    right: 2px;
+                }
+            }
+        }
+
+        &:hover {
+            border: 1px solid rgba(34, 123, 60, 0.75);
+        }
+
+        .MuiDropzoneArea-textContainer {
+            display: flex;
+            align-items: center;
+            justify-content: center;
+            margin-top: 50px;
+
+            p {
+                font-size: 12px;
+                color: rgba(255, 255, 255, 1);
+            }
+
+            .MuiDropzoneArea-icon {
+                width: 24px;
+                height: 16px;
+                color: #FFFFFF;
+                ;
+            }
+        }
+    }
+
+    .googlmap-wrape {
+        height: 330px;
+        width: 100%;
+        margin-top: 89px;
+
+        .search-wrap {
+            position: relative;
+
+            .search-input {
+                border: none;
+                border-bottom: 1px solid #599C6D;
+                width: 39%;
+                margin-bottom: 20px;
+                outline: none;
+                padding-right: 20px;
+            }
+
+            ::placeholder {
+                color: #599C6D;
+                opacity: 1;
+                /* Firefox */
+            }
+
+            :-ms-input-placeholder {
+                /* Internet Explorer 10-11 */
+                color: #599C6D;
```

Comment:
Can we move all the colors to a separate `colors.scss` file and use them by importing ?

Reply:
This comment applicable to all other places where we hard coded the colors

Reply:
If these styles are reused in multiple components then we can put them in Theme.js. @gangadharjannu , @codelover1110 and I discussed a convention forward to put styling. See section "Element styling":
https://docs.google.com/document/d/1wudInNCCeCvXGYRXtkXkxdMZuGTuAmvDTcJmpAzZPhY/edit#heading=h.amuyuc6b5lmt